### ISSUE TITLE
no indication of coach content during import

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#7354]
### Observed behavior

channels have coach content, but there is no indicator on the main channel listing

| channel list | channel details |
|--|--|
|![image](https://user-images.githubusercontent.com/2367265/88209204-34bf2180-cc07-11ea-8ed5-0ee1e59caa00.png) | ![image](https://user-images.githubusercontent.com/2367265/88209265-4ef8ff80-cc07-11ea-8895-b29b6072f7f4.png) |

### Expected behavior

should be possible to scan for coach content in channel list

### User-facing consequences

not as much information available as desired

### Context

0.14.0

**Comment by riddhiavlani** [comment]
@jtamiace @khangmach 
1. The coach content icon is visible on the channel list only if such resources have been imported from the channel. It shows the icon + the no. of resources. Its not visible if coach facing resources aren't imported. Check image below: 
![image 7.png](https://images.zenhubusercontent.com/6054cbf6e1e4e5d5abc9e647/18101d0e-bae0-4444-b124-5e07e70caee6)

2. We can add just the coach content icon and a corresponding tooltip to all channels that have coach content (and haven't been imported). Check screen shot. However do you think this may cause confusion with the screenshot above. Users might think that in the screen shot above, there’s only 1 coach resource available in the entire channel, which could be misleading
![Coach resources indicated on channel list (3).png](https://images.zenhubusercontent.com/6054cbf6e1e4e5d5abc9e647/43494b63-b041-4c6d-844b-087a8089dc2d)

**Comment by indirectlylit** [comment]
That count as implemented looks quite misleading. It says "contains 1 coach resource", but what it means to say is "1 coach resource on disk" which is not nearly as useful.

A simple change would be to show "Contains coach resources" without any count, anytime the channel has any coach resources whether they're currently imported or not.

**Comment by riddhiavlani** [comment]
Yeah sounds good to me - We don't show the count and just keep the icon (screenshot 2) for channels that have coach facing content

**Comment by marcellamaki** [comment]
The decision here is that when selecting resources for import, 

> show "Contains coach resources" without any count, anytime the channel has any coach resources whether they're currently imported or not

Visually, it should display as shown in screenshot #2

**Comment by adibmbrk** [comment]
Hey @marcellamaki, can you assign this issue to me please?

**Comment by AlexVelezLl** [comment]
Hi @adibmbrk! I will assign this to you. Thank you! Please, let us know if you have any question 👐.

**Comment by adibmbrk** [comment]
Hi All, Just an update on this issue, I was able to reproduce this locally. I have a question regarding this, does this change need to be be done for all the available languages?

**Comment by MisRob** [comment]
Hi @adibmbrk, I would expect the icon and tooltip be visible no matter of the language. The only criteria is what @marcellamaki mentioned [here](https://github.com/learningequality/kolibri/issues/7354#issuecomment-2654586703).

If you're asking because of the tooltip text, I recommend studying [the Internationalization page in the developer documentation](https://kolibri-dev.readthedocs.io/en/develop/i18n.html#internationalization). Then you can try to search whether this string already exists or if you need to add a new one.

**Comment by lucascoelho-unl** [comment]
Hi @adibmbrk are you still working on this issue? If not, I would love to give it a try! If you are still working then all the best, and I'm open to help if needed. :)

**Comment by adibmbrk** [comment]
Hey @MisRob @lucascoelho-unl, I'm still working on this issue. It's taking a bit longer as I'm preparing my GSoC proposal, but rest assured, I'll get it done soon.

**Comment by adibmbrk** [comment]
Hey @marcellamaki @MisRob, 

When selecting resources for import, we use the `GET /api/content/remotechannel` endpoint to fetch channel data from Studio.

If we want to display the "Contains coach resources" icon and tooltip when selecting a channel to import, the response from this endpoint would need to include a `num_coach_contents field`, so we can identify which channels have coach content. This approach will include making changes in the studio repo as well.

Is this the approach you had in mind?? or have i just misunderstood the requirement

Let me know what you think

**Comment by adibmbrk** [comment]
Hi @marcellamaki @MisRob, just a kind reminder on this issue, I'd appreciate it if you could take a look and let me know if the above is the right approach.

**Comment by rtibbles** [comment]
Hi @adibmbrk - yes, I think you're correct that it would need broader changes to the public channel APIs. As such, I suspect this is not appropriate for community contribution and needs more planning. I'll unassign you and remove the label. Thanks!
